README.md

```diff
@@ -45,9 +45,16 @@ To delete all key-value pairs in the Tags field of a specific HANA instance:
 ```
 az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --set tags={}
 ```
+
+To enable monitoring for a specific HANA instance:
+
+```
+az hanainstance monitor --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --hana-vnet $HANA_VNET  --hana-hostname $HANA_HOSTNAME --hana-instance-num $HANA_INSTANCENUM --hana-db-username $HANA_DB_USERNAME --hana-db-password $HANA_DB_PW
```

Comment:
I may have missed this, but was the method initially not "enable monitoring" or "toggle monitoring"?

Reply:
Also, what exactly is the instance name and why is it the only HANA-specific parameter that is not prefixed by "HANA"?

Reply:
instance name is the name of the `hanainstance` since it is used in the other methods as `instance-name` I kept it that way for consistency. It may make sense to remove the prefix from the other arguments so that the user does not have to type in such long names